src/main/java/com/develhope/spring/deals/models/Order.java

```diff
@@ -27,6 +26,9 @@ public class Order {
     @ManyToOne(fetch = FetchType.LAZY)
     private User user;
 
+    @ManyToOne(fetch = FetchType.LAZY)
+    private User seller;
```

Comment:
Deve essere nullable true altrimenti non funziona più nulla nel momento in cui non lo passi. Difatti te hai modificato i vecchi test, invece i vecchi devono funzionare così come sono, poi devi farne di nuovi dove controlli che funzioni tutto anche se passi il seller.

Reply:
Ma a conti fatti, come fai a unire una cosa che non esiste a qualcosa che esiste?